x-pack/plugins/security/public/management/role_mappings/role_mappings_grid/role_mappings_grid_page.tsx

```diff
@@ -500,7 +498,7 @@ export class RoleMappingsGridPage extends Component<Props, State> {
   };
 
   private onRoleMappingsDeleteCancel = () => {
-    this.tableRef.current?.setSelection([]);
+    this.setState({ selectedItems: [] });
   };
```

Comment:
I checked with the team, and we'd like to take this as an opportunity to change the behavior here. None of the other grid pages deselect the active selection on delete/bulk operation cancel. We'd like to remove the `onRoleMappingsDeleteCancel` handler here.

Reply:
Is there any chance y'all could change this after this PR merges in a separate PR? Since this touches multiple teams, and it's also blocking an EUI deprecation, I'd ideally like to keep it fairly minimal and not risk any tests/types that I don't totally understand changing (it looks like this would affect another `DeleteProvider` component as well). I'd be happy to help with that separate PR if needed later!

Reply:
Sure, Cee. No problem!

Reply:
Separate follow-up PR: https://github.com/elastic/kibana/pull/175838